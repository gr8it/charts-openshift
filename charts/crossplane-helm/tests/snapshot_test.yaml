suite: Snapshot test
templates:
  - templates/*
values:
  - ../values.example.yaml
release:
  name: crossplane-helm
  namespace: apc-crossplane-system

tests:
  - it: manifests should match snapshot
    asserts:
      - matchSnapshot: {}
