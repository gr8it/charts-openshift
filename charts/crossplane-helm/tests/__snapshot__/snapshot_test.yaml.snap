manifests should match snapshot:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: NetworkPolicy
    metadata:
      labels:
        app.kubernetes.io/instance: crossplane-helm
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: crossplane-helm
        app.kubernetes.io/version: 1.19.2
        helm.sh/chart: crossplane-helm-1.5.0
      name: crossplane-helm-allow-proxy
      namespace: apc-crossplane-system
    spec:
      egress:
        - to:
            - ipBlock:
                cidr: 1.2.3.4/32
            - ipBlock:
                cidr: 1.2.3.5/32
      podSelector: {}
      policyTypes:
        - Egress
